app/web/i18n/namespaces.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Namespaces single-source-of-truth
+ *
+ * This file must be kept in CommonJS syntax (not Typescript or ES6) because it
+ * is imported from the next-i18next.config.js that is executed as plain
+ * NodeJS code.
+ */
+const AUTH = "auth";
+const COMMUNITIES = "communities";
+const CONNECTIONS = "connections";
+const DASHBOARD = "dashboard";
+const DONATIONS = "donations";
+const GLOBAL = "global";
+const MESSAGES = "messages";
+const PROFILE = "profile";
+const SEARCH = "search";
+
+const NAMESPACES = {
+  AUTH,
+  COMMUNITIES,
+  CONNECTIONS,
+  DASHBOARD,
+  DONATIONS,
+  GLOBAL,
+  MESSAGES,
+  PROFILE,
+  SEARCH,
+};
```

Comment:
In fact, do we even need all of the namespaces in an object anywhere in our code at the moment? Since they are also individually exported again below, I don't think we need this?

Reply:
Oh... you are right. Maybe YAGNI, but it seems to me that having it exported in both formats (object and array) but defined just once wasn't a waste of time and might well be used in some places like we just tried to do in the resources file. It is also more like a pattern I like to follow. Maybe we could remove the lists from the exports section at all, but I see it like different "endpoints" for accessing namespaces (individual, array and object)

Reply:
The list is being used in the `next-i18next` config though, whereas the object usage I guess is TBD. Should we keeping code if it's YAGNI? :p

Probably one more small thing on naming... should the keys be all uppercase too? i.e. `NAMESPACES.AUTH` instead of `NAMESPACES.auth`? I guess I haven't seen many object properties in uppercase unless it's some magic value. But maybe you could argue in this case the constant is a magic value.

Reply:
Done